examples/simulation_examples/airplane_case.py

```diff
@@ -0,0 +1,57 @@
+import flow360 as fl
+from flow360 import units as u
+
+
+def createBaseParams_airplane():
```

Comment:
`create_base_params_airplane`

also, why do we want this to be in the examples? We already have example on how to create Flow360Params. We can use this as second example. But other examples should not refer to this file. If they need to load example config, it needs to be loaded from file. Every example should be standalone runnable script

Reply:
fixed.